composer.json

```diff
@@ -29,7 +29,8 @@
         "symfony/process": "^2.3|^3.0"
     },
     "require-dev": {
-        "psr/log": "^1.0"
+        "psr/log": "^1.0",
+        "phpunit/phpunit": "^4.8"
```

Comment:
It's unrelated to you PR, and there is already a PR to add it which has not been merged.

https://github.com/gitonomy/gitlib/pull/98